README.md

```diff
@@ -829,6 +829,42 @@ export class ReactiveFormComponent implements OnInit {
 }
 ```
 
+### React
+
+React controlled components naturally keep some internal mutable state and update it based on user input via `setState()`.
```

Comment:
Could you write that there are two available approaches to forms: using controlled and uncontrolled input ? An that the former is recommended.